aliases.js

```diff
@@ -21,20 +21,37 @@
 // Enables ES2015 import/export in Node.js
 
 // Registers an alias for this module
-const path = require('path');
+const {resolve} = require('path');
+const fs = require('fs');
 
-const ALIASES = {
-  'deck.gl/test': path.resolve(__dirname, './test'),
-  'deck.gl': path.resolve(__dirname, './src'),
-  'deck.gl-layers': path.resolve(__dirname, './src/experimental-layers/src'),
-  'deck.gl-test-utils': path.resolve(__dirname, './src/test-utils/src')
-};
+const MODULE_PATH = resolve(__dirname, './modules');
 
-if (module.require) {
-  module.require('reify');
+// Get all module info
+const MODULES = {};
```

Comment:
This is an intimidating start to the file. Things that can help:
* Create some well named function for this.
* Place that function at the end of the file, after a `// HELPER FUNCTIONS` comment, just calling it here.
* spend some time on improving comments, e.g`Get all module info` doesn't quite get me started on the right track. `Build a map containing the contents of 'package.json' for all deck.gl modules` might be better.